src/components/KeyboardShortcuts/index.tsx

```diff
@@ -115,6 +162,11 @@ export class KeyboardShortcutsBase extends React.Component<Props> {
   }
 
   render() {
+    const { compareInfo } = this.props;
+    const disabledClassname = makeClassName({
+      [styles.disabled]: !compareInfo,
```

Comment:
This could be:

```ts
const disabledClassname = !compareInfo ? styles.disabled : '';
```

It would be a little easier to read if it were specific to the item (not the state):

```ts
const diffKeysClassName = !compareInfo ? styles.disabled : '';
```